get.py

```diff
@@ -13,13 +13,16 @@
 
 args = parser.parse_args()
 
-try:
-    output = requests.get(args.url).text
-except requests.exceptions.MissingSchema:
-    exit(
-        f"Invalid URL '{args.url}': No scheme supplied. Perhaps you meant https://{args.url}?")
-except requests.exceptions.ConnectionError:
-    exit(f"404: URL not found")
+
+def set_output():
+    try:
+        return requests.get(args.url).text
```

Comment:
The other way to do this instead of except handling is to `beginswith()` and then add it if that fails.